cmd/common.go

```diff
@@ -1,35 +1,11 @@
 package cmd
 
 import (
-	"fmt"
-
 	"github.com/jbrekelmans/kube-compose/pkg/config"
-	"github.com/urfave/cli"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
-
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-const (
-	environmentIDFlagName = "env-id"
-	namespaceFlagName     = "namespace"
-)
-
-func GlobalFlags() []cli.Flag {
-	return []cli.Flag{
-		cli.StringFlag{
-			Name:   environmentIDFlagName + ", e",
-			EnvVar: "KUBECOMPOSE_ENVID",
```

Comment:
.GetEnv("KUBECOMPOSE_ENVID") or use Viper to be able to get same value

Reply:
Ditto to all `EnvVar` values

Reply:
Done